2_process.R

```diff
@@ -16,6 +17,39 @@ p2_targets_list <- list(
                        sites = p1_drb_temp_sites_sf)
   ),
   
+  tar_target(p2_buffered_nhd_reaches,
```

Comment:
I'm wondering whether we need `p2_buffered_nhd_reaches` and associated downstream targets? In terms of the buffered depth to bedrock, we'll only need those values for the area around the NHM segment since we're only computing the additional groundwater features at the NHM scale for now. It's _possible_ that we would add the extra groundwater features for the full NHD network down the line, but for simplicity we could probably omit these for now. What do you think?